### PR TITLE
Update @yandex-cloud/uikit to v2.3.1 [with breaking changes fix]

### PR DESCRIPTION
**Pull request by ariser**
Changelog for v2: https://github.com/yandex-cloud/uikit/blob/main/CHANGELOG_V2.md

`ydb-ui-components` updated to uikit v2, this modifies code base to match `@yandex-cloud/uikit@^2`, for the following update of `ydb-ui-components` to `v2`

### CODE REVIEW

**Comment by tsufiev** [comment]
@ariser It's better to set the type of the last commit to 'fix', this way it will produce a new (patch) version, otherwise it won't. If you do not want to produce a new version of embedded-ui-components with a new uikit version, it's fine to use 'refactor' type.

File: src/containers/Tenant/ObjectGeneral/ObjectGeneral.scss [hunk 1]
```diff
@@ -16,23 +16,22 @@
         padding: 13px 20px 0;
 
         box-shadow: inset 0 -1px 0 0 var(--yc-color-line-generic);
-        @include body3-typography();
```
Comment (ariser) [inline src/containers/Tenant/ObjectGeneral/ObjectGeneral.scss:19]: now achieved via size prop

File: src/containers/Tenant/ObjectGeneral/ObjectGeneral.scss [hunk 2]
```diff
@@ -16,23 +16,22 @@
...
-        display: flex;
+        // fix for bug in uikit, gap is set for yc-tabs__item:not(:last-child),
+        // it doesn't work for wrapped items
+        // feel free to remove if the bug is fixed
+        &:not(:last-child) {
+            margin-right: var(--yc-tabs-gap);
+        }
     }
 
-    &__tab-label {
```
Comment (ariser) [inline src/containers/Tenant/ObjectGeneral/ObjectGeneral.scss:32]: was needed for custom icons, which are now supported natively

File: src/containers/Node/Node.scss [hunk 3]
```diff
@@ -20,7 +20,6 @@
 
     &__tabs {
         padding: 16px 20px 0;
-        @include body2-typography();
```
Comment (ariser) [inline src/containers/Node/Node.scss:23]: now achieved via size prop

**Review by Raubzeug — DISMISSED** [review]

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**